wiki/Tournaments/TNMT/1/en.md

```diff
@@ -55,6 +55,109 @@ All prizes are sponsored by **![][flag_CN] [hans1999](https://osu.ppy.sh/users/6
+  - [Middleisland - Zap \[Electric SV\]](https://osu.ppy.sh/b/1909358)
+  - [Frums - Wavetapper \[Stage 1: Memory Factory\]](https://osu.ppy.sh/b/1811863)
+- Tiebreaker
+  - **[Camellia - farewell to today \[Another\]](https://osu.ppy.sh/b/1769258)**
+
+### Round of 16
+
+- FreeMod
+  - [Primastella - Koigokoro \[Kokoro\]](https://osu.ppy.sh/b/2110798)
+  - [Ani - Ververg (feat. B) \[Usagi's Lv.12,1\]](https://osu.ppy.sh/b/708608)
+  - [DJ Siesta - Another Day \[riu's Serenity (edited)\]](https://osu.ppy.sh/b/2147167)
+  - [antiPLUR - Runengon \[Hard\]](https://osu.ppy.sh/b/2034201)
+  - [Umeboshi Chazuke - Bison Charge \[MEGA's Hard\]](https://osu.ppy.sh/b/1755410)
+  - [w_tre - Shannon's Theorem \[Medium\]](https://osu.ppy.sh/b/1519707)
+  - [RevenG - Sakura \[1:37:00 AMCherry Blossom [NO SLOWJAMS]\]](https://osu.ppy.sh/b/2116953)
+  - [aaaa - carnation \[fascination\]](https://osu.ppy.sh/b/1392215)
+  - [SOUND HOLIC Vs. VENUS feat. Nana Takahashi - Musou \[EXHAUST\]](https://osu.ppy.sh/b/987971)
+  - [Post Malone - Money Made Me Do It \[yuh\]](https://osu.ppy.sh/b/2104840)
+  - [Sakuraburst - Serotonin \[Transmission Problem [SV]\]](https://osu.ppy.sh/b/1382422)
+- Tiebreaker
+  - **[Ryo Arue - Forza \[Sophistication (edit)\]](https://osu.ppy.sh/b/2148054)**
+
+### Group Stage
+
+- FreeMod
+  - [Ichi sato bocchi (Morishita Chisaku) - Ne, issho ni kaero. \[Chill\]](https://osu.ppy.sh/b/2110884)
+  - [fripSide - black bullet (TV Size) \[Normal Curse\]](https://osu.ppy.sh/b/2069199)
+  - [M2U - Quo Vadis \[Hard\]](https://osu.ppy.sh/b/2018240)
+  - [Artisan - Summon \[Voyage\]](https://osu.ppy.sh/b/2066947)
+  - [Kamex - Collapse Remix \[Hard\]](https://osu.ppy.sh/b/1796039)
+  - [The Flashbulb - Black Lawn Finale \[Hayden's Lv.12 NSV\]](https://osu.ppy.sh/b/1253922)
+  - [sasakure.UK - Merry Christmas, Mr.Lawrence \[Hard\]](https://osu.ppy.sh/b/2009260)
+  - [void - Valedict \[Hard\]](https://osu.ppy.sh/b/1306452)
+  - [rainlord. - i wonder. \[hard.\]](https://osu.ppy.sh/b/2087790)
+- Tiebreaker
+  - **[sakuzyo - After Altale \[End of a Journey\]](https://osu.ppy.sh/b/1220666)**
+
+### Qualifiers
+
+- [Mr. Fantastic - Hyperactive \[Stage 1: Subtle\]](https://osu.ppy.sh/b/2130420)
+- [Tennyson - Pancake Feet \[Stage 2: Quick Steps\]](https://osu.ppy.sh/b/2130186)
+- [Eve - As You Like It \[Stage 3: Regulation\]](https://osu.ppy.sh/b/2130198)
+- [Silentroom - NULCTRL \[Stage 4: THE CLUB\]](https://osu.ppy.sh/b/2130237)
+
+## Match Results
+
+### Round of 16
+
+| Player | Score | Score | Player |
+| --: | :-: | :-: | :-- |
+| Maou | 0 | **5** | **NestleiFan** |
+| **Davvyforte** | **5** | 0 | NutAndGo |
+| **Yukina-** | **5** | 4 | Antalf |
+| **Halo-** | **5** | 0 | vincent4399 |
+| edisonhello | 2 | **5** | **Horou** |
+| BITONIC | 0 | **5** | **Ladylag** |
+| HD_AdreNaline | 1 | **5** | **palmEuEi** |
+| EDTA233 | 2 | **5** | **mewmew** |
+
+### Group Stage
+
+|  |  |  |  |  |  |  |  |  |  |  |
```

Comment:
if you want to have the table structured like this, split it into two tables and put the column titles in the table header
```md
| Group A | Score |  | Group B | Score | ...
| :-: | :-: | :-: | :-: | :-: | ...
| **Maou** | **12** | ...

| Group E | Score | ...
| :-: | :-: | ...
| **Yukina-** | **12** | ...